runtimes/bundles/org.jboss.tools.rsp.server.minishift/src/main/java/org/jboss/tools/rsp/server/minishift/discovery/MinishiftVersionLoader.java

```diff
@@ -23,8 +23,7 @@
 	public static final String VERSION_KEY2 = "minishift";
 	public static final String CDK_VERSION_KEY = "CDK";
 	public static final String CDK_VERSION_KEY_OLD = "CDK Version";
-	public static final String CRC_VERSION_KEY = "CRC";
-	public static final String CRC_VERSION_KEY_OLD = "CRC Version";
+	public static final String CRC_VERSION_KEY = "version";
```

Comment:
When doing crc version, the version key for the crc version is simply "version".  Since crc is new, there is no "old" version key either =P